docs/source/openshift_sdn/README.md

```diff
@@ -0,0 +1,66 @@
+This is openshift-sdn, the default network plugin for OpenShift (both
+OKD and OCP). It uses Open vSwitch to connect pods locally, with VXLAN
+tunnels to connect different nodes.
+
+OpenShift SDN is designed to be installed by the [OpenShift Network
+Operator](https://github.com/openshift/cluster-network-operator), and
+certain components of it (such as the Deployment and DaemonSet
+objects) are found there.
+
+This module defines two images: the sdn image, which contains OpenShift SDN
+(both controller and node components) and the kube-proxy image, which is
+deployed by the Network Operator for third-party network plugins that need it.
+(Kube-proxy is built from here rather than from the origin repo so
```

Comment:
If there previous sentence is finished with a "." I don't think we actually need the parenthesis

Reply:
ditto